Source/RpmNextGen/Public/Api/Characters/Models/CharacterCreateResponse.h

```diff
@@ -12,4 +12,18 @@ struct RPMNEXTGEN_API FCharacterCreateResponse : public FApiResponse
 
 	UPROPERTY( EditAnywhere, BlueprintReadWrite, Category = "Ready Player Me", meta = (JsonName = "data"))
 	FRpmCharacter Data;
+
+	FCharacterCreateResponse()
+	{		
+	}
+
+	FCharacterCreateResponse(FRpmCharacter Data)
+	{
+		this->Data = Data;
+	}
+
+	bool IsValid() const
+	{
+		return !Data.Id.IsEmpty() && !Data.GlbUrl.IsEmpty();
```

Comment:
GlbUrl.IsEmpty? 😄 Was there a case? I think this is mandatory field

Reply:
Well it's partially used to determine whether the request failed to trigger loading from cache. But it's a fair point probably the use of the bWasSuccessful bool is enough :D 
![image](https://github.com/user-attachments/assets/f48c0a51-d68a-4d1f-a524-53e5b5b0b76d)
The only other way it might be useful is if somebody was doing something custom and forgets to set the id or url, so we could prevent missuse of it, probably it's overkill though